### DISCUSSION TITLE
What is a frame?

### DISCUSSION THREAD

**Discussion by NastyFlytrap** [uBlockOrigin/uBlock-issues#3040] [Q&A] [answered]
### Prerequisites

- [X] I verified that this is not a filter list issue. Report any issues with filter lists or broken website functionality in the [uAssets issue tracker](https://github.com/uBlockOrigin/uAssets/issues).
- [X] This is NOT an issue with [YouTube](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [Facebook](https://www.github.com/uBlockOrigin/uAssets/issues/3367) or [Twitch](https://www.github.com/uBlockOrigin/uAssets/issues/5184).
- [X] This is not a support issue or a question. For support, questions, or help, visit [/r/uBlockOrigin](https://www.reddit.com/r/uBlockOrigin/).
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uBlock-issues/issues?q=is%3Aissue) to avoid opening a duplicate issue.
- [X] The issue is not present after disabling uBO in the browser.
- [X] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I am reporting is not normal behavior.

### I tried to reproduce the issue when...

- [X] uBO is the only extension.
- [X] uBO uses default lists and settings.
- [X] using a new, unmodified browser profile.

### Description

In the context of ublock origin, what does 3rd party frame blocking mean?

Obviously, it blocks 'frames' from domains that i am not visiting right now, but what is a frame in this context? Like a data packet? Or a div?

I tried looking through the wiki but that doesnt explain what a frame is, only how ublock deals with them. 

### A specific URL where the issue occurs.

doomworld.com

### Steps to Reproduce

I wanted to check miniwad and the youtube video was a 'frame' and on other sites unblocking youtube would let it play but not here. For some reason on this site i also had to unblock 3rd party frame blocking

### Expected behavior

 A

### Actual behavior

 A

### uBO version

 A

### Browser name and version

 A

### Operating System and version

A

**Comment by gorhill** [comment]
> - [x] This is not a support issue or a question. For support, questions, or help, visit [/r/uBlockOrigin](https://www.reddit.com/r/uBlockOrigin/).

Do not ignore what is asked from you in the prerequisites.

**Reply by NastyFlytrap** [reply]
Im sorry

**Comment by ghost** [comment, marked as answer]
Frame is a website embedded in other website. Often video players on various pages are YouTube frames.

**Reply by krystian3w** [reply]
`<frame>`/`<iframe>`, `<object>` or `<embed>`

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/object = https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_object_html
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/embed = https://www.w3schools.com/tags/tryit.asp?filename=tryhtml5_embed_html
